app/controllers/socure_webhook_controller.rb

```diff
@@ -4,6 +4,17 @@ class SocureWebhookController < ApplicationController
   skip_before_action :verify_authenticity_token
 
   def create
-    render json: { message: 'Got here.' }
+    if token_valid?
+      render json: { message: 'Secret token is valid.' }
+    else
+      render status: :unauthorized, json: { message: 'Invalid secret token.' }
+    end
```

Comment:
Will we be doing more validation in the future? Should we start doing [form validation pattern](https://github.com/18F/identity-idp/blob/main/docs/backend.md#forms-formresponse-analytics-and-controllers)?

```suggestion
    form = SocureWebhookValidationForm.new
    result = form.submit(headers: request.headers)
    if result.success?
      render json: { message: 'Secret token is valid.' }
    else
      render status: :unauthorized, json: { message: result.first_error_message }
    end
```

Reply:
I don't think we will be; my understanding is that we'll use the webhook just for status updates on submitted requests, and the secret token is just an anti-ddos measure.

Certainly, if we do end up using information from the webhook, then we should follow the form validation pattern we already have.